oss.gg/7_create_a_template.md

```
@@ -40,4 +40,6 @@ Your turn 👇
 » 11-October-2024 by Prabin Subedi [Starter template for Unkey Route protecttion in  Hono Webframework in Cloudflare workers ](https://github.com/prabincankod/hono-unkey-cflare)
 
 » 10-October-2024 by Harsh Bhat [FastAPI Unkey Boilerplate](https://github.com/harshsbhat/unkey-fastapi-boilerplate)
+
+» 14-October-2024 by Shrey **[Fastify Route Ratelimiting With Unkey Boilerplate](https://github.com/shreykx/unkey-fastify)**.
```

Markdownlint

Punctuation: '.'
Trailing punctuation in heading

(MD026, no-trailing-punctuation)